Article/Article.js

```diff
@@ -3,17 +3,26 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = document.querySelectorAll('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+
```

Comment:
Nice, you could've also done.   this.domElement.querySelector('.expandButton');
    this.expandButton.innerText = 'expand';